docs/whats-new/dotnet-2020-04.md

```diff
@@ -0,0 +1,124 @@
+---
+title: ".NET docs: What's new for April 2020"
+description: "What's new in .NET docs for April 2020."
+ms.date: 04/27/2020
+---
+
+# .NET docs: What's new for April 2020
+
+Welcome to what's new in .NET docs for April 2020. This article lists some of the major changes to docs during this period.
+
+## Architecture guides
+
+### New articles
+
+- [gRPC](../architecture/cloud-native/grpc.md) - Cloud Native - Chapter 5 - Updates to gRPC
+
+### Updated articles
+
+- [App configuration](../architecture/blazor-for-web-forms-developers/config.md) - Blazor ebook config
+
+## .NET Core
+
+### New articles
+
+- [Trim self-contained deployments and executables](../core/deploying/trim-self-contained.md) - Create trimming doc
+- [Ubuntu 20.04 Package Manager - Install .NET Core](../core/install/linux-package-manager-ubuntu-2004.md) - Install for Ubuntu 20.04
+
+## C# language
+
+### New articles
+
+- [Reserved attributes: Determine caller information](../csharp/language-reference/attributes/caller-information.md) - Reorganize and update language reference for attributes processed by the C# compiler, including nullable analysis
+- [Reserved attributes: ConditionalAttribute, ObsoleteAttribute, AttributeUsageAttribute](../csharp/language-reference/attributes/general.md) - Reorganize and update language reference for attributes processed by the C# compiler, including nullable analysis
+- [Reserved attributes: Assembly level attributes](../csharp/language-reference/attributes/global.md) - Reorganize and update language reference for attributes processed by the C# compiler, including nullable analysis
+- [Reserved attributes contribute to the compiler's null state static analysis](../csharp/language-reference/attributes/nullable-analysis.md) - Reorganize and update language reference for attributes processed by the C# compiler, including nullable analysis
+- [Nullable reference types (C# reference)](../csharp/language-reference/builtin-types/nullable-reference-types.md) - Update language reference for nullable token and related topics
+
+### Updated articles
```

Comment:
The [Structure types](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/struct) article has got a lot of updates in March and in April. I agree those have been mostly consolidations, not new material. Only wonder if the omission of that article is be design or by accident.

Reply:
That's by accident. I'll add it. Thanks for pointing this out.